README.md

```diff
@@ -1,2 +1,147 @@
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+  - [👤**Authors** ](#authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 Leaderboard App<a name="about-project"></a>
+
+**Leaderboard app* is a web app that allows you to add your name and scores to the leaderboard. 
+
+Here are some features of the page:
+
+Add your name and score to the leaderboard.
+
+
+## 🛠 Built With <a name="built-with"></a>
+    HTML
+    CSS
+    Javascript
+    Webpack
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+none
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+- Create a local directory that you want to clone the repository.
+
+- On the terminal navigate to the folder you created and run this command: git clone https://github.com/Kashif-Saleem-Ghuman/LeaderBoard-by-Kashif.git
+
+- Now navigate to the repository folder using command prompt cd LeaderBoard-by-Kashif.
+
+- Make sure to intall linters using this link: https://github.com/microverseinc/linters-config.
+
+-Make sure to add project dependancies ( webpack)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Prerequisites
+
+In order to run this project you need:
+
+-Google chrome
+
+-Internet Explorer
+
+-Any broswer
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- Open the command prompt/terminal and navigate to the created directory.
+
+- On the terminal run this command git clone https://github.com/Kashif-Saleem-Ghuman/LeaderBoard-by-Kashif.git.
+
+- Go to the repository folder using command prompt cdTo-LeaderBoard-by-Kashif.
+
+
+- Make sure to intall linters using this link: https://github.com/microverseinc/linters-config.
+
+- Make sure to intall all dependencies.
+
+
+--->
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👤**Authors** <a name="authors"></a>
+<h4>Kashif Saleem</h4>
+
+- GitHub: [@githubhandle](https://github.com/Kashif-Saleem-Ghuman)
+- Twitter: [@twitterhandle](https://twitter.com/Kashif14Saleem)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/kashif-saleem-45ba95215/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Implement some UX improvements: To dynamically add and remove names and scores using the Leaderboard API.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Kashif-Saleem-Ghuman/To-do-list-by-Kashif/issues).
```

Comment:
- [ ] **[OPTIONAL]:** Please consider changing this link to the Leaderboard one! 🤓